cypress/e2e/tests/pages/explorer/node-list.spec.ts

```diff
@@ -0,0 +1,53 @@
+import ClusterDashboardPagePo from '@/cypress/e2e/po/pages/explorer/cluster-dashboard.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import { dummyNode } from '@/cypress/e2e/blueprints/explorer/nodes';
+
+describe('Nodes list', { tags: ['@adminUser'], testIsolation: 'off' }, () => {
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+
+    // Add dummy node that used ot cause a problem
+
+    // cy.deleteRancherResource('v1', 'nodes', 'bigip1');
+    cy.createRancherResource('v1', 'nodes', dummyNode);
+  });
+
+  after(() => {
+    cy.deleteRancherResource('v1', 'nodes', 'bigip1');
+  });
+
+  it('should show the nodes list page', () => {
+    ClusterDashboardPagePo.navTo();
+
+    const nav = new ProductNavPo();
+
+    nav.navToSideMenuEntryByLabel('Nodes');
+
+    cy.contains('.title > h1', 'Nodes').should('be.visible');
+
+    const nodeList = new BaseResourceList(cy.get('[data-testid="cluster-list-container"]'));
+
+    // Wait for loading indicator to go
+    cy.get('.data-loading').should('not.exist');
+
+    // Check table has 2 tows
+    nodeList.resourceTable().sortableTable().rowElements({ timeout: 2500 }).should((rows: any) => {
+      expect(rows).not.to.equal(undefined);
+      expect(rows).to.have.length(2);
+    });
+
+    // Check the node names
+    nodeList.self().find('td.col-link-detail > span > a').then((links: any) => {
```

Comment:
cypress/e2e/po/components/sortable-table.po.ts `rowNames`  should give you the same as this

Reply:
That didn't work for me - I think we have issues with that to do with the chaining and retries. I didn't want to change what we already had. There's something strange going on we could look at.

Reply:
Done - needed to use a string selector